ui/src/containers/VisualiseForm/NewVisualisation/TemplateCardList.js

```diff
@@ -6,6 +6,12 @@ import { default as TemplateMostPopularVerbsCard } from 'ui/containers/Visualisa
 import { default as TemplateMostPopularActivitiesCard } from 'ui/containers/Visualisations/TemplateMostPopularActivities/Card';
 import { default as TemplateMostActivePeopleCard } from 'ui/containers/Visualisations/TemplateMostActivePeople/Card';
 import { default as TemplateWeekdaysActivityCard } from 'ui/containers/Visualisations/TemplateWeekdaysActivity/Card';
+import { default as TemplateCuratrInteractionsVsEngagementCard } from 'ui/containers/Visualisations/TemplateCuratrInteractionsVsEngagement/Card';
+import { default as TemplateCuratrCommentCountCard } from 'ui/containers/Visualisations/TemplateCuratrCommentCount/Card';
+import { default as TemplateCuratrLearnerInteractionsByDateAndVerbCard } from 'ui/containers/Visualisations/TemplateCuratrLearnerInteractionsByDateAndVerb/Card';
+import { default as TemplateCuratrUserEngagementLeaderboardCard } from 'ui/containers/Visualisations/TemplateCuratrUserEngagementLeaderboard/Card';
+import { default as TemplateCuratrProportionOfSocialInteractionsCard } from 'ui/containers/Visualisations/TemplateCuratrProportionOfSocialInteractions/Card';
+import { default as TemplateCuratrActivitiesWithMostCommentsCard } from 'ui/containers/Visualisations/TemplateCuratrActivitiesWithMostComments/Card';
```

Comment:
Do we need to `import { default }`? I think we should be able to do the following.
```js
import TemplateCuratrActivitiesWithMostCommentsCard from 'ui/containers/Visualisations/TemplateCuratrActivitiesWithMostComments/Card';
```

Reply:
No. sorry, I'll fix them.

Reply:
Okay thank you 👍